### PR TITLE
added description to filezilla.json

### PR DESCRIPTION
**Pull request by gautamkrishnar**
added description

### CODE REVIEW

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "filezilla_cheat_sheet",
     "name": "FileZilla",
+    "description": "FileZilla is a powerful free FTP client for transferring files over the Interne",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: - typo at the end
- let's have this describe the cheat sheet content

**Comment by gautamkrishnar** [comment]
@moollaza :+1:

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "filezilla_cheat_sheet",
     "name": "FileZilla",
+    "description": "FileZilla is a powerful free FTP client for transferring files over the Internet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/filezilla.json]: @gautamkrishnar we can start with "A powerful..."

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/filezilla.json]: why not ?

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/filezilla.json]: @moollaza what did you say?

Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: Ya, let's avoid redundancy. No need to reiterate the name of the subject.

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/filezilla.json]: ok

**Comment by gautamkrishnar** [comment]
@moollaza updated :+1:

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:q

**Pull request merged**